Electrical/Pinouts.md

```diff
@@ -201,7 +201,7 @@ is 10156001-051100LF.
 | 43  | FPR_CTRL    | See [4]                      |
 | 44  | SWITCH      | Power button pin             |
 | 45  | FPR_LED_R   | Low-side control by the EC   |
-| 46  | FPR_LED_W   | Low-side control by the EC   |
+| 46  | FPR_LED_B   | Low-side control by the EC   |
```

Comment:
hey! not affiliated with Framework, but this pin is actually, indeed, FPR_LED_W - as in, it controls a white LED on the fingerprint reader, not a blue one.

Reply:
That seems to differ from the schematic which has it labelled as +FPR_LED_B_Q, are you sure there is a Red, Green and White LED and not a Red, Green, and Blue LED?

Reply:
absolutely, we just checked it on a physical Framework laptop! the pinouts diagram is more up-to-date when it comes to what the pins actually stand for

Reply:
Yep, W is correct.  The B in the schematic is out of date.  We actually had RGB early in development and moved to RGW because the uniformity looked better.

Reply:
Ack, undid the change and added a note to mention the deviation